tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,239 @@
+package rbac
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+
+type rbTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	standardUser       *management.User
+	standardUserClient *rancher.Client
+	session            *session.Session
+	cluster            *management.Cluster
+}
+
+
+func (rb *rbTestSuite) TearDownSuite() {
+	rb.session.Cleanup()
+}
+
+func (rb *rbTestSuite) SetupSuite() {
+	testSession := session.NewSession(rb.T())
+	rb.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(rb.T(), err)
+
+	rb.client = client
+
+	//Get cluster name from the config file and append cluster details in rb
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(rb.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(rb.client, clusterName)
+	require.NoError(rb.T(), err, "Error getting cluster ID")
+	rb.cluster, err = rb.client.Management.Cluster.ByID(clusterID)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *rbTestSuite) ValidateClustersList(client *rancher.Client, expectedClusterCount int, clusterID string) (err error) {
+	clusterList, err := client.Provisioning.Cluster.List(&types.ListOpts{}) 
+
+	// assert the count of clusters obtained as the user is same as the expected cluster count
+	//assert cluster list contains right cluster ID
+	if err == nil{
+		actualResult := len(clusterList.Data)
+		assert.Equal(rb.T(), expectedClusterCount, actualResult)
+		if actualResult > 0 {
+			if clusterID != "" {
+				assert.Equal(rb.T(), clusterID, clusterList.Data[0].Status.ClusterName )
+			}
+		}
+	}
+	return err
+}
+
+
+func (rb *rbTestSuite) ValidateClusterMembers(role string) {
+
+	//Testcase1 Verify if cluster members are able to list clusters
+	fmt.Printf("Testcase1 - Validating the cluster count obtained as a %s. \n",role)
+	err := rb.ValidateClustersList(rb.standardUserClient, 1, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	fmt.Println("Validation successful!")
```

Comment:
Remove all the fmt.Println statements and use the logging instead. Either rb.T().Log or Logrus.Info(). No Printlns at all